src/CodeStyle/Core/Analyzers/Options/AnalyzerConfigOptionsExtensions.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.CodeAnalysis.Options;
+
+namespace Microsoft.CodeAnalysis
+{
+    internal static class AnalyzerConfigOptionsExtensions
```

Comment:
>AnalyzerConfigOptionsExtensions [](start = 26, length = 31)

I presume this will go away after the compiler's AnalyzerConfigOptions type is available? Do we have more such files that are temporary? If so, might want to file a tracking issue that lists all temporary files to be removed.